spec/factories/items.rb

```diff
@@ -2,9 +2,9 @@
 
 FactoryGirl.define do
   factory :item do
-    donor_description      { Faker::Lorem.sentence }
-    state                  'submitted'
-    reject_reason { Faker::Lorem.sentence }
+    donor_description { Faker::Lorem.sentence }
+    state             'submitted'
```

Comment:
Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.